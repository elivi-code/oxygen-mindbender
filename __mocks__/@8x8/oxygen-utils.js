export * from '@8x8/oxygen-utils';

const useFontSize = jest.fn(() => 16);

export { useFontSize };
