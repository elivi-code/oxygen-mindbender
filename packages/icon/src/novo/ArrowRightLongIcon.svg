<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M16.7197 15.9697C16.4268 16.2626 16.4268 16.7374 16.7197 17.0303C17.0126 17.3232 17.4874 17.3232 17.7803 17.0303L22.2803 12.5303C22.5732 12.2374 22.5732 11.7626 22.2803 11.4697L17.7803 6.96967C17.4874 6.67678 17.0126 6.67678 16.7197 6.96967C16.4268 7.26256 16.4268 7.73744 16.7197 8.03033L19.9393 11.25L2.19643 11.25C1.8118 11.25 1.5 11.5858 1.5 12C1.5 12.4142 1.8118 12.75 2.19643 12.75L19.9393 12.75L16.7197 15.9697Z" fill="#292929"/>
</svg>
