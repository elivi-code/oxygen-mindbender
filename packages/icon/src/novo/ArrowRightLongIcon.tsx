import React from 'react';
import type { IconProps } from '../utils/createSvgIcon';
import createSvgIcon from '../utils/createSvgIcon';

export type { IconProps };
export default createSvgIcon(<g><path d="M16.7197 15.9697a.75.75 0 0 0 0 1.0606.75.75 0 0 0 1.0606 0l4.5-4.5a.75.75 0 0 0 0-1.0606l-4.5-4.5a.75.75 0 0 0-1.0606 1.0607l3.2196 3.2196H2.1964c-.3846 0-.6964.3358-.6964.75s.3118.75.6964.75h17.7429l-3.2196 3.2197Z"/></g>, 'ArrowRightLongIcon');
