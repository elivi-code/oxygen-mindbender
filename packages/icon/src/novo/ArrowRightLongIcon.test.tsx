/* eslint-disable */
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { icon as theme } from '@8x8/oxygen-constants';
import ArrowRightLongIcon from './ArrowRightLongIcon';

describe('<ArrowRightLongIcon>', () => {
  let wrapper;

  function setup(props = {}) {
    return renderer.create(<ArrowRightLongIcon theme={theme} {...props} />).toJSON();
  }

  it('should render correctly', () => {
    wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
/* eslint-enable */
