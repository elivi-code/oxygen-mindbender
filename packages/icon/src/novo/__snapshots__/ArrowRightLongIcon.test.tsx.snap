// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ArrowRightLongIcon> should render correctly 1`] = `
.c0 {
  height: 24px;
  width: 24px;
  min-height: 24px;
  min-width: 24px;
  -webkit-user-select: auto;
  -moz-user-select: auto;
  -ms-user-select: auto;
  user-select: auto;
  pointer-events: auto;
  fill: #1C2025;
}

<svg
  className="c0"
  color={null}
  disabled={false}
  role="presentation"
  selected={false}
  size={null}
  viewBox="0 0 24 24"
>
  <g>
    <path
      d="M16.7197 15.9697a.75.75 0 0 0 0 1.0606.75.75 0 0 0 1.0606 0l4.5-4.5a.75.75 0 0 0 0-1.0606l-4.5-4.5a.75.75 0 0 0-1.0606 1.0607l3.2196 3.2196H2.1964c-.3846 0-.6964.3358-.6964.75s.3118.75.6964.75h17.7429l-3.2196 3.2197Z"
    />
  </g>
</svg>
`;
